packages/devtools_app/lib/src/screens/debugger/breakpoint_manager.dart

```diff
@@ -138,6 +138,16 @@ class BreakpointManager with DisposerMixin {
     }
   }
 
+  void _saveAndClearCurrentBreakpoints() {
+    if (breakpointsWithLocation.value.isNotEmpty) {
+      _previousIsolateBreakpoints
+        ..clear()
+        ..addAll(_breakpointsWithLocation.value);
+    }
+    _breakpoints.value.clear();
+    _breakpointsWithLocation.value.clear();
```

Comment:
this won't notify the listeners of the ValueNotifiers. these should instead be `.value = []`

Reply:
Oh good to know, done!